kiwigrid-sidecar/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM kiwigrid/k8s-sidecar:0.1.157 as builder
+
+FROM python:3.9.0b3-alpine3.12
+
+ENV         PYTHONUNBUFFERED=1
+WORKDIR     /app
+RUN echo "kubernetes==10.0.0" >> requirements.txt
```

Comment:
why not copying the requirements from the original image as well? It might change over time

Reply:
I see, it got removed from the image :(

Reply:
maybe let's download it from the repo directly?

Reply:
> maybe let's download it from the repo directly?

used version 0.1.157 is not available as release download, latest release was 0.1.144